pos_partner_firstname/static/src/js/PartnerDetailsEdit.js

```diff
@@ -45,7 +45,7 @@ odoo.define("pos_partner_firstname.PartnerDetailsEdit", function (require) {
                         processedChanges.lastname === ""
                     ) {
                         return this.showPopup("ErrorPopup", {
-                            title: _t("Both Customer First and Last Name Are Required"),
+                            title: _t("Both customer first name and last name are required."),
```

Comment:
are you sure on this one ? 

https://www.deepl.com/translator#fr/en/Le%20nom%20et%20le%20pr%C3%A9nom%20du%20client%20sont%20obligatoires

Reply:
This is involved in v17 here https://github.com/OCA/pos/pull/1126